Media/UrlList/MediaUrlList - Twitch.as

```diff
@@ -74,8 +85,10 @@ array<dictionary> GetChunkOfUsersOnline(string allFollowersIds, string header) {
 			//Set every online channel in list of urls.
 			for (int k = 0, lenNames = streams.size(); k < lenNames; k++) {
 				string isPlaylist = streams[k]["type"].asString();
-				string viewers = streams[k]["viewer_count"].asString();
-				string login = streams[k]["user_name"].asString();
+				string viewers = streams[k]["viewer_count"].asString();				
```

Comment:
@jeongminkim-islab Could you remove the blank spaces at the end of the line?

Reply:
Okey. I'll fix that.

Reply:
Okay, nevermind, I'll do it in the next commit.